### PR TITLE
[JENKINS-35341][JENKINS-35343] Delete selectable Bundle

### PR DESCRIPTION
**Pull request by jpthiery**
List Bundle files to display bundle files.
Then allow to Delete them selecting from UI.
Done thanks to help of @alecharp .

### CODE REVIEW

**Comment by alecharp** [comment]
here my :+1:. We will deal with the UI later. cc @aheritier @christ66

File: src/main/java/com/cloudbees/jenkins/support/SupportAction.java [hunk 1]
```diff
@@ -47,12 +47,10 @@
...
 
+import java.io.File;
+import java.io.FilenameFilter;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (christ66) [inline src/main/java/com/cloudbees/jenkins/support/SupportAction.java]: :NIT: would prefer to not have wildcards for import statements.

**Comment by jpthiery** [comment]
I had changed import as asked.

**Comment by jpthiery** [comment]
ping ?

**Comment by christ66** [comment]
LGTM.

**Pull request merged**